opensrp-app/src/main/java/org/smartregister/domain/jsonmapping/Column.java

```diff
@@ -7,5 +7,8 @@
 public class Column {
     public String column_name;
     public String type;
+    public String data_type;
+    public String source_format;
+    public String save_format;
```

Comment:
Could this use camelCase convention and use @SerializedName to map to json keys that may have _